packages/rolldown/src/utils/bindingify-output-options.ts

```diff
@@ -28,6 +29,11 @@ export function bindingifyOutputOptions(
     globals,
     file,
   } = outputOptions
+
+  if (typeof dir === 'string' && typeof file === 'string') {
```

Comment:
Could you move this validation to rust side?

Reply:
It's difficult at now, here need to refactor the `NormalizedBundlerOptions#dir` to `Option<String>` at first.

Reply:
Yes.